### ISSUE TITLE
About the L_1 norm gradient update

### ISSUE THREAD

**Issue by cmhcbb** [JunfengGo/AEVA-Blackbox-Backdoor-Detection-main#1]
Hi,
   I couldn't find the l1 gradient normalization claimed in the paper's appendix to do the update. The attack code looks identical to HSJA l2 version. Could you point out where you have made the change?
  Thanks!

**Comment by JunfengGo** [comment]
Hi, Minhao, I very much appreciate your interest in our work, AEVA and I am also a fan of your work OPT/Sign-OPT.  You are right, we directly use L2 version of HSJA.  However, I think it doesn't matter since our work is to select the singularity pixels no matter which norm is used. You can also use L1 norm version of HSJA and change the threshold for anomaly index!

**Comment by cmhcbb** [comment]
Hi Junfeng,
     Thanks. It clears my confusion. However, if you use the l2 norm of HSJA, is there any trick or special selection of hyperparamter to make the perturbation as sparse as possible? Or are you just selecting those pixels with the largest value change?

**Comment by JunfengGo** [comment]
Hi Minhao，

I don't think apply any tricks in the l2 perturbation。 As you see that， i
just leverage HSJA to craft l2 perturbation
